src/rime/config/config_data.cc

```diff
@@ -66,12 +66,15 @@ bool ConfigData::LoadFromFile(const path& file_path, ConfigCompiler* compiler) {
   modified_ = false;
   root.reset();
   if (!std::filesystem::exists(file_path)) {
-    LOG(WARNING) << "nonexistent config file '" << file_path << "'.";
+    LOG(WARNING) << "nonexistent config file '" << file_path.u8string() << "'.";
     return false;
   }
-  LOG(INFO) << "loading config file '" << file_path << "'.";
+  LOG(INFO) << "loading config file '" << file_path.u8string() << "'.";
   try {
-    YAML::Node doc = YAML::LoadFile(file_path.string());
+    std::ifstream fin(file_path.c_str());
```

Comment:
Is it possible to just pass `c_str()` to `YAML::LoadFile`?

Reply:
YAML::LoadFile accept std::string only, so I don't think c_str() is OK.